pkg/apiserver/webhooks/datavolume-validate.go

```diff
@@ -222,6 +222,14 @@ func (wh *dataVolumeValidatingWebhook) validateDataVolumeSpec(request *v1beta1.A
 	}
 
 	accessModes := spec.PVC.AccessModes
+	if len(accessModes) == 0 {
```

Comment:
is this part of this change?

Reply:
Yes it is, so this one is kind of interesting. Before in our code schema we specified that accesModes is required, and this the len cannot be 0. However the PVC.spec doesn't say it is actually required (omitempty like before), and since we take the spec from the kubernetes code we have to follow it. So now it is no longer required. So this means I can make a PVC spec without accessModes. Kubernetes will actually fail that in a validation web hook, and thus so should we. This change is for that reason.